### PR TITLE
Added Description to package.json

### PR DESCRIPTION
**Pull request by lelouchB**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "coding-blocks.github.io",
   "version": "1.0.0",
-  "description": "",
+  "description": "Official Website of codingblocks.com, a programming bootcamp based in Delhi",
```
Comment (sarthak-1998) [inline package.json]: of Coding Blocks 

a Programming Boot-camp for beginner to advanced.

**Comment by sarthak-1998** [comment]
Please make the desired changes

**Comment by lelouchB** [comment]
> Please make the desired changes

Have made the changes

**Comment by lelouchB** [comment]
@sarthak-1998 please review

**Comment by sarthak-1998** [comment]
@witty123 Please verify.

**Comment by lelouchB** [comment]
@sarthak-1998  I am closing this PR and I will write to hacktoberfest to not allow codingblocks to act as an organization in upcoming hacktoberfest.

**Pull request closed without merging**